### PR TITLE
Secure fpga weebhook

### PR DESCRIPTION
**Pull request by bart0sh**
This PR changes webhook deployment to run its container with non-root user privileges.

### CODE REVIEW

File: scripts/webhook-deploy.sh [hunk 1]
```diff
@@ -54,6 +62,8 @@ done
 [ -z ${kubectl} ] && kubectl="kubectl"
 [ -z ${mode} ] && mode="preprogrammed"
 [ -z ${namespace} ] && namespace="default"
+[ -z ${uid} ] && uid="$(id -u)"
```
Comment (mythi) [inline scripts/webhook-deploy.sh]: would some standard non-login user be better default? or print a warning if uid/gid are not set?

Comment (bart0sh) [reply scripts/webhook-deploy.sh]: done

File: scripts/webhook-deploy.sh [hunk 2]
```diff
@@ -54,6 +62,8 @@ done
 [ -z ${kubectl} ] && kubectl="kubectl"
 [ -z ${mode} ] && mode="preprogrammed"
 [ -z ${namespace} ] && namespace="default"
+[ -z ${uid} ] && uid="1000"
+[ -z ${gid} ] && gid="1000"
```
Comment (mythi) [inline scripts/webhook-deploy.sh]: I was thinking something like `$(id -u nobody)` (or some other suitable standard user that works)

Comment (bart0sh) [reply scripts/webhook-deploy.sh]: I'm sorry, I forgot to add creating fpga user. Now it should be done.

File: scripts/webhook-deploy.sh [hunk 3]
```diff
@@ -36,6 +36,14 @@ while [[ $# -gt 0 ]]; do
             namespace="$2"
             shift
             ;;
+	--uid)
+            uid="$2"
```
Comment (mythi) [inline scripts/webhook-deploy.sh]: would it be good to remind user to update Dockerfile too if the default is overridden?

Comment (bart0sh) [reply scripts/webhook-deploy.sh]: I tend to remove this option and use hardcoded uid and gid. It seems to create more problems than solves.
@mythi wdyt?

Comment (mythi) [reply scripts/webhook-deploy.sh]: +1

File: scripts/webhook-deploy.sh (hunk 2 above)
Comment (bart0sh) [reply scripts/webhook-deploy.sh]: done

File: scripts/webhook-deploy.sh (hunk 3 above)
Comment (bart0sh) [reply scripts/webhook-deploy.sh]: done

**Pull request merged**